angular-standalone/src/app/login.service.ts

```diff
@@ -1,15 +1,14 @@
 import { Injectable } from '@angular/core';
 import { HttpClient, HttpResponse } from '@angular/common/http';
-import { catchError, filter, map } from 'rxjs/operators';
-import { Observable, of, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+import { Observable, of } from 'rxjs';
 
 @Injectable()
 export class LoginService {
   constructor(private readonly http: HttpClient) { }
 
   login(username: string, password: string): Observable<{ message: string, status: number}> {
     return this.http.post<HttpResponse<string>>('/auth', { username, password }).pipe(
-      filter(res => res.status === 200),
```

Comment:
The result of a normal Angular http request returns the json parsed body, not the full request. It assumes a normal status so the filter doesn't make sense here.

Reply:
Good catch! Apparently my RxJs skills are becoming dull

Reply:
I debugged this for a long time so I forgot this as well!!